app/main.py

```diff
@@ -2,0 +1,16 @@
+def copy_file(command: str) -> None:
+    splited = command.split(" ")
+    if len(splited) != 3 or splited[0] != "cp":
+        return None
+
+    original_file = splited[1]
+    copied_file = splited[2]
+
+    if original_file == copied_file:
+        return None
+
+    with (
+        open(original_file, "r") as file_in,
+        open(copied_file, "w") as file_out
```

Comment:
Consider adding error handling for file operations. If the file does not exist or there are permission issues, the program will raise an exception. Using a try-except block can help manage such errors gracefully.